src/lib/journey/_utilities/step.utilities.ts

```diff
@@ -61,12 +61,7 @@ export function initCheckValidation() {
  * @returns {boolean}
  */
 export function shouldRedirectFromStep(step: FRStep) {
-  try {
-    step.getCallbackOfType(CallbackType.RedirectCallback);
-    return true;
-  } catch (e) {
-    return false;
-  }
+  return step.getCallbacksOfType(CallbackType.RedirectCallback).length > 0;
```

Comment:
It's really strange how using `getCallbacksOfType` now works for me - in the previous PR I was getting weird errors when using this so I used `getCallbackOfType` in a try catch. Anyway, it seems to be working how I would have originally expected it to now, which is nicer.